test/apis/README.md

```
@@ -0,0 +1 @@
+Refer to https://github.com/openshift/api/tree/master/tests for more details.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)